ocaml-lsp-server/src/document.ml

```diff
@@ -27,7 +27,7 @@ module Syntax = struct
     | id -> failwith ("Unexpected language id " ^ id)
 end
 
-let { Logger.log } = Logger.for_section "ocaml-lsp-server"
+(* let { Logger.log } = Logger.for_section "ocaml-lsp-server" *)
```

Comment:
Why are we commenting this stuff out?

Reply:
Ah I forgot to clean that up
The logger is only used here for the logging of the whole new text every time a document changes. Which 90% of the "over-logging" we do.
I think the server is stable enough to remove this now.

Reply:
Yeah, I never found this message to be overly useful either.